lua/nvim-treesitter/indent.lua

```diff
@@ -192,13 +192,23 @@ end
 
 local indent_funcs = {}
 
+local function autocmd_is_defined(event, pattern, command_str)
+  return vim.api.nvim_exec(string.format("au %s %s", event, pattern), true):find(command_str) ~= nil
```

Comment:
We definitely want to avoid string manipulation and calling Vimscript functions here. If necessary (which I don't think it is; see my other comment), use `nvim_get_autocommands`.

Reply:
I still think the other approach is much better, though.